libs/librepcb/libraryeditor/sym/dialogs/symbolpinpropertiesdialog.cpp

```diff
@@ -55,6 +55,9 @@ SymbolPinPropertiesDialog::SymbolPinPropertiesDialog(SymbolPin& pin,
   mUi->spbPosY->setValue(mSymbolPin.getPosition().getY().toMm());
   mUi->spbRotation->setValue(mSymbolPin.getRotation().toDeg());
   mUi->spbLength->setValue(mSymbolPin.getLength()->toMm());
+      
```

Comment:
It seems that the GitHub webinterface doesn't show them, but my local Git client says there are whitespaces on this line. Would be nice to have them removed ;)

Reply:
Done.